docs/language/values-and-types.mdx

```diff
@@ -168,6 +168,20 @@ Integers have multiple built-in functions you can use.
   answer.toString()  // is "42"
   ```
 
+- `cadence•fun fromString(_ input: String): T?`
```

Comment:
Unlike the other functions documented here, this is not an instance function, but a "static" function on the type itself. Maybe document it as:

```suggestion
- `cadence•fun T.fromString(_ input: String): T?`
```

And document that `T` is any integer type.

Maybe also separate this from the remaining instance functions by moving it to a separate "section" below, e.g.

"All integer types support the following functions:"